scipy/linalg/_sketches.py

```diff
@@ -8,6 +8,7 @@
 import numpy as np
 
 from scipy._lib._util import check_random_state
+import scipy.sparse
```

Comment:
you can instead use `from scipy.sparse import csr_matrix` to keep things a bit more concise.

Reply:
thanks, did that